### DISCUSSION TITLE
How to exlude some fields from /admin/{modelname}/create endpoint

### DISCUSSION THREAD

**Discussion by bralbral** [aminalaee/sqladmin#428] [General]
Hello!

Could you please explain what options can exclude some columns from the create form?

I want to exclude the "CreatedAt" and "Groups" fields.

![image](https://user-images.githubusercontent.com/54455457/218170300-fc34d2f9-cfa8-4022-9754-2789242db28c.png)

**Comment by aminalaee** [comment]
Hey,
have you checked here https://aminalaee.dev/sqladmin/configurations/#form?
form_columns or form_excluded_columns?

**Reply by bralbral** [reply]
Oh, sorry to disturb you. I should read the documentation more carefully. Thanks for the quick response!
